orchestration/src/main/java/com/sap/ai/sdk/orchestration/OrchestrationResponse.java

```diff
@@ -0,0 +1,49 @@
+package com.sap.ai.sdk.orchestration;
+
+import com.sap.ai.sdk.orchestration.client.model.ChatMessage;
+import com.sap.ai.sdk.orchestration.client.model.CompletionPostResponse;
+import com.sap.ai.sdk.orchestration.client.model.TokenUsage;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import javax.annotation.Nonnull;
+import lombok.RequiredArgsConstructor;
+import lombok.val;
+
+public record OrchestrationResponse(
```

Comment:
Debatable if this should be a record or not. The AllArgs constructor is a bit awkward. Also, we might offer more convenience later on..